src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/MethodologyRepository.cs

```diff
@@ -21,6 +21,7 @@ public MethodologyRepository(ContentDbContext contentDbContext)
         public async Task<List<Methodology>> GetByPublication(Guid publicationId)
         {
             return await _contentDbContext.PublicationMethodologies
+                .AsQueryable()
```

Comment:
I think it's already Queryable at this point?

Reply:
Without the `AsQueryable`, there is an ambiguious invocation error on the `Where`.

Reply:
I checked each instance in the comments below, and they all are necessary, so I've resolved those comments.

Reply:
Cool ta, thanks for checking!